exercises/robot-simulator/canonical-data.json

```diff
@@ -55,13 +55,14 @@
       "cases": [
         {
           "description": "changes the direction from north to east",
-          "property": "turnRight",
+          "property": "instructions",
```

Comment:
What do you think about making the property a command, maybe something like `move`

Reply:
Should it be plural, `moves`?  Since the instructions may contain many moves.

Reply:
I don't think so, it's a command. You are telling the robot to `move` and providing it some instructions.